database/seeders/HighlightSeeder.php

```diff
@@ -9,6 +9,7 @@ class HighlightSeeder extends Seeder
 {
     public function run(): void
     {
+        // XXX I don't think this is necessary anymore. In the original migration a default was set for this field.
```

Comment:
Yes, this is not needed anymore. This was a one-time migration to catch up all existing records to the behavior of the new functionality.